extension/content_script.js

```diff
@@ -247,6 +247,9 @@ function highlight(tracker) {
 	// Notice trackerStyle here is NOT immediately updated when user changes settings;
 	// We need the old trackerStyle name to be able to find and remove the styling, 
 	// since the next sentence will get a new style. 
+	let displaySettings = settingsView.getSettings();
+	let trackerStyle = "trackerHighlighter"+displaySettings[1]+"Shadow"+displaySettings[2]; // 1st element is highlighter type, 2nd element is shadow type
```

Comment:
i highly suggest creating a small class for this, in settings_view.js, just like how doc.js also has sentencepointer

class SentencePointer {
    constructor(containerId, start, end) {
        this.containerId = containerId;
        // Marks the sentence offset within the container. start is inclusive, end exlusive.
        this.start = start;
        this.end = end;
    }
}

It might not seem much right now, but this will keep piling up and be harder to refactor later on if u add more settings.

Reply:
Agree

Reply:
Thanks for thinking of that!

Reply:
This is addressed in a new PR, so I'm going to sqash+merge